src/cms-content/learn/products-landing.json

```diff
@@ -1,24 +1,27 @@
+      "buttons": []
+    },
+    {
+      "productName": "COVID Response Simulator",
+      "productId": "covid-response-simulator",
+      "productSubtitle": "Allows you to forecast scenarios",
+      "productDescription": "Try it [here](https://docs.google.com/spreadsheets/u/1/d/1iUQvuY-DGbuxGBDrj1fvaDjk-peunHm89sMJe0z24U8/copy?usp=sharing).\n\nThe COVID Response Simulator is a localized, spreadsheet version of the public Covid Act Now (CAN) model. With it, you can take a powerful [SEIR epidemiology model](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology#The_SEIR_model) and customize it for your county to help plan your response to COVID. The inputs and assumptions in the simulator are modifiable and can be changed to reflect your local realities.\n\nIn addition, you can project the impact of specific Non-Pharmaceutical Interventions (NPIs) by adjusting customizable levels of public compliance and closure for:\n\n* Schools and universities\n* Large events\n* Bars and restaurants\n* Offices and factories\n* Houses of worship\n* Personal care\n* Nonessential retail\n* Essential retail\n* Entertainment\n* Outdoor recreation\n\nYou’ll also be able to simulate the impact of different levels of compliance with mask mandates and shelter-in-place orders, as well as modify the demographic distribution of your county’s population. Based on your inputs, the simulator generates data and graphs illustrating COVID forecasts with and without these NPIs, including estimated case numbers and hospitalizations.\n\n![Example view of the COVID Response Simulator](/images_cms/crs-screenshot.png \"CRS\")\n\n> ### Get started\n>\n> 1. Before you begin, you can watch a full tutorial (30 minutes) on the simulator [here](https://youtu.be/6lYf9ry1DWE) or a shorter tutorial (10 minutes) on how to use the simulator specifically for modeling NPIs [here](https://youtu.be/xiI_kC8hGTA).\n> 2. Click on [this link](https://docs.google.com/spreadsheets/u/1/d/1iUQvuY-DGbuxGBDrj1fvaDjk-peunHm89sMJe0z24U8/copy?usp=sharing) to create a copy of the simulator for your use. You’ll be prompted to make a copy so that you can work in your own document.\n> 3. Review the information in the first tab in Orange labeled “About”. Then continue onward to the second tab in Green labeled “Location and Inputs Setup”.\n\n*If you have any questions about our model or the data we use, reach out to us at [info@covidactnow.org](mailto:info@covidactnow.org). You can also learn more on our [YouTube channel](https://www.youtube.com/channel/UChs298932BKWanjwWLVUO4w).*"
```

Comment:
It's probably safer to link to `https://covidactnow.org/covid-response-simulator` here because it would be difficult to check that the link is up to date every time we update the CRS (we change the link each time)